### PR TITLE
WIP: support for ActiveSupport::Notifications

### PR DESCRIPTION
**Pull request by timoschilling**
This PR is still work in progress, feedback and feature requests are welcome!

This PR adds some use full informations to the rails log:
1___rails_command_server__ruby_

### CODE REVIEW

File: lib/trailblazer/rails/logging.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        define_method method do |*args, &block|
+          ActiveSupport::Notifications.instrument "#{method}.trailblazer", operation: self, method: method do
+            super *args, &block
+          end
+        end
+      end
+    end
+
+    Operation.class_eval do
+      prepend Trailblazer::Logging::Instrumentation
```
Comment (apotonick) [inline lib/trailblazer/rails/logging.rb]: Cool, but do we really need `prepend`? I'd rather include it properly in the railtie?

**Comment by fernandes** [comment]
hahahha really cool @timoschilling !! :+1:

File: lib/trailblazer/rails/logging.rb (hunk 1 above)
Comment (timoschilling) [inline lib/trailblazer/rails/logging.rb]: No, it was the first way that has worked, but maybe you can help to improve it

**Comment by seuros** [comment]
@timoschilling do you have time to wrap this feature ? I really like it.

**Comment by marctich** [comment]
@apotonick this never got done, but it would be a great addition

**Pull request closed without merging**